types/final-form-set-field-touched/final-form-set-field-touched-tests.ts

```diff
@@ -0,0 +1,3 @@
+import setFieldTouched from 'final-form-set-field-touched';
+
+setFieldTouched; // $ExpectType Mutator<object>
```

Comment:
Generally a more functional test is better as opposed to just testing the name of a type.